bin/node/src/chain_spec.rs

```diff
@@ -322,12 +283,11 @@ struct AccountsConfig {
 }
 
 /// Provides accounts for GenesisConfig setup based on distinct staking accounts.
-/// Assumes validator, controller and stash are all separate accounts.
-fn configure_distinct_staking_accounts(
+/// Assumes validator == stash, but controller is a distinct account
```

Comment:
oh, that's a twist for me - I would definitely put this assumption to the chainspec-generating command documentation in order to make sure that user is aware of this

Reply:
Let me add some docs about it

Reply:
okay, you have also put it in the ADR so it's okay with me now